File: Ejercicio2.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

struct Alumno {
    string nombre;
    string calificacion;

    string toString() const {
        return nombre + " " + calificacion;
    }
};

int main() {
    vector<Alumno> alumnos = {
        {"Pedro", "20"},
        {"Alejandro", "22"},
        {"Carlos", "18"}
    };

    cout << "Alumnos + Calificación:" << endl;
    for (int i = 0; i < alumnos.size(); i++) {
        cout << "[" << i << "]: " << alumnos[i].toString() << endl;
    }
    return 0;
}

File: Ejercicio2.cs
using system;
using system.Collections.Generic;

class Alumno
{
    public string Nombre { get; set; }
    public string Calificacion { get; set; }

    public Alumno(string nombre, string calificacion)
    {
        Nombre = nombre;
        calificacion = Calificacion;
    } 

    public override string ToString() {
        return Nombre + " " + Calificacion;
    }
}

List<Alumno> alumnos = new List<Alumno> {
    new Alumno("Pedro", "20"),
    new Alumno("Alejandro", "22"),
    new Alumno("Carlos", "18")
};

Console.WriteLine("Alumnos + Calificación:");
for (int i = 0; i < alumnos.Count; i++) {
    Console.WriteLine("[" + i + "]: " + alumnos[i]);
}